test/fake_modules/package_import_map/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "imports":{
+    "#simple": "simple-resolved",
+    "#subpath": "subpath-resolved/subpath",
+    "#conditional": {
+      "node": "conditional-node",
+      "types": "@types/conditional"
+    },
+    "#multi-conditional": {
+      "node": {
+        "test": "conditional-node-test"
```

Comment:
Same question, what is the `"test"` condition?

Reply:
In our project we add some custom conditions by using `enhanced-resolve` where we can specify whatever condition names we want. So in tests, we have `test` condition set, in storybook we set `storybook`.  This allows some imports to be redirected according to the environment.

Reply:
I see, and node/other tools just ignore conditions they don't know about right?

Reply:
Yeah basically any tool treats the conditions as a set and if it finds an object it descends into the first key that is in that set.  If it finds a string, it restarts resolution replacing the first path component with that string and maybe doing wildcard replacement if any.  If it hits a dead end, the import fails.